Core/include/Acts/Fitter/detail/VoidKalmanComponents.hpp

```diff
@@ -26,9 +26,9 @@ struct VoidKalmanComponents {
   ///
   /// @return void-calibrated measurement
   template <typename measurement_t, typename parameters_t>
-  Result<measurement_t> operator()(measurement_t m,
-                                   const parameters_t& /*pars*/) const {
```

Comment:
Does it actually complain about this comment?

Reply:
?

Reply:
I readded the commented parameter names.

Reply:
Saw it now, sorry.